### PR TITLE
Added transactions with type 'Credit'

### PR DESCRIPTION
**Pull request by adoerrer**

### CODE REVIEW

**Comment by NancyBolduc** [comment]
Thanks for adding the transaction type of "Credit" to this table.

**Review by NancyBolduc — APPROVED** [review]
Thanks for adding the Transaction Type of 'Credit' to this table.

**Review by sbeltaine — APPROVED** [review]
This change is approved.

File: sql/derived_tables/finance_transaction_invoices.sql [hunk 1]
```diff
@@ -40,7 +40,8 @@ FROM
     LEFT JOIN organization_organizations AS oo ON json_extract_path_text(ii.data, 'vendorId') = oo.id
 WHERE
     transaction_type = 'Pending payment'
-    OR transaction_type = 'Payment';
+    OR transaction_type = 'Payment'
+    OR transaction_type = 'Credit';
 
 CREATE INDEX ON folio_reporting.finance_transaction_invoices (transaction_id);
```
Comment (sbeltaine) [inline sql/derived_tables/finance_transaction_invoices.sql:47]: This change is approved

**Review by sbeltaine — APPROVED** [review]
This change is approved

**Pull request merged**